lib/elasticsearch.js

```diff
@@ -96,6 +96,8 @@ function insert(listCounters, listTimers, listTimerData, listGaugeData) {
   if (payload.length === 0) {
     // No work to do
     return;
+  } else {
+    payload = payload.concat(["\n"]);
```

Comment:
https://www.elastic.co/guide/en/elasticsearch/reference/5.5/docs-bulk.html
ES bulk API requires payload to end with newline. 

It seemed like this was the best place to add it in, since it adds it into the empty check for payload.
Seems like the `\n` could be directly added to the payload right before it gets written to (https://github.com/DispatchBot/statsd-elasticsearch-backend/pull/1/files#diff-ce7a9282ae0091392bb9fb3e7d518f9dR150), but that would probably include another empty check?
Overall, it seemed cleaner to do it here, but not sure if there's a better alternative in terms of performance.